CONTRIBUTING.md

```diff
@@ -0,0 +1,102 @@
+# Contributing Guidelines
+
+The `F0.Minesweeper` open source project is hosted on **GitHub**, so we start this document off with the [GitHub Community Guidelines](https://help.github.com/en/github/site-policy/github-community-guidelines).
+
+## Coding Style
+See [.editorconfig](./source/.editorconfig) file.
+
+See also
+* [EditorConfig.org](https://editorconfig.org/)
+* [Visual Studio Docs](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/code-style-rule-options)
+
+## Development Techniques
+Apply _test-driven development_ when adding features or fixing bugs.
+Follow the rules of _Clean Code_ for the entire codebase.
+
+## Version Scheme
+See [Semantic Versioning](https://semver.org/).
+
+Although games do not declare a public API, `MAJOR`, `MINOR` and `PATCH` changes for each release are observed from the player's point of view.
+
+## Issues
+See [GitHub Guides: Issues](https://guides.github.com/features/issues/).
+
+Each _issue_ should be submitted with an intention-revealing _title_ and a descriptive _comment_.
+Define the scope of an _issue_ by applying meaningful _labels_ rather than prefixing the _title_.
+Before work is started, the _issue_ should have contributing _assignees_ added or updated.
+
+_Issues_ that result in changes to the repository are closed via [pull requests](#pull-requests).
+
+## Branching Workflow
+Based on the [GitHub flow](https://guides.github.com/introduction/flow/).
+
+The [main](https://github.com/Flash0ver/F0.Minesweeper) branch is the _default_ branch.
+It includes all verified changes being published in the next release.
+New _topic branches_ are created off of `main`.
+Tags target the `main` branch.
+
+The [publish](https://github.com/Flash0ver/F0.Minesweeper/tree/publish) branch is the second long-running branch.
+It represents the state of the _latest release_.
+On each release published, the `publish` branch is updated from `main` and pushed to.
+
+Every change to the repository is committed to short-lived _topic branches_.
+New _topic branches_ are based on `main`.
+Their work in progress is status checked, reviewed and discussed through _pull requests_.
```

Comment:
```suggestion
Their work in progress is status checked, reviewed and discussed through [pull requests](#pull-requests).
```
We've got the link to the specific section in all other usages so we should add it here as well.